sources/osg/Geometry.js

```diff
@@ -315,8 +318,7 @@ Geometry.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( No
         if ( !this._primitives.length ) return;
 
         if ( this._extVAO === undefined && Geometry.enableVAO ) { // will be null if not supported
-            var extVAO = WebGLCaps.instance( state.getGraphicContext() ).getWebGLExtension( 'OES_vertex_array_object' );
-            this._extVAO = extVAO;
+            this._extVAO = WebGLCaps.instance( state.getGraphicContext() ).getWebGLExtension( 'OES_vertex_array_object' );
```

Comment:
Probably getWebGLExtension should be renamed because we dont do the difference between supporting an extension and having the feature in the core (webgl2).